BenchmarkDotNet.IntegrationTests.FSharp/project.json

```diff
@@ -0,0 +1,50 @@
+{
+  "version": "1.0.0-*",
+
+  "commands": {
+    "testOnlyFromConsole": "xunit.runner.dnx"
+  },
+
+  "dependencies": {
+    "BenchmarkDotNet": {
+      "target": "project"
+    },
+    "xunit": "2.1.0",
+    "xunit.core": "2.1.0"
+  },
+
+  "frameworks": {
+    "net45": {
```

Comment:
This isn't necessary for test projects. When running it with DNX, you'll only need the `dnx*` TFMs :smile: